src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/sykefraværshistorikk/SykefraværshistorikkController.java

```diff
@@ -102,17 +118,48 @@
         SummertSykefraværshistorikk summertSykefraværshistorikkVirksomhet =
                 summertSykefraværService.hentSummertSykefraværshistorikk(
                         underenhet,
-                        new ÅrstallOgKvartal(2021, 2),
+                        SISTE_PUBLISERTE_ÅRSTALL_OG_KVARTAL,
                         antallKvartaler
                 );
 
         SummertSykefraværshistorikk summertSykefraværshistorikkBransjeEllerNæring =
                 summertSykefraværService.hentSummertSykefraværshistorikkForBransjeEllerNæring(
                         underenhet,
-                        new ÅrstallOgKvartal(2021, 2),
+                        SISTE_PUBLISERTE_ÅRSTALL_OG_KVARTAL,
                         antallKvartaler
                 );
 
         return Arrays.asList(summertSykefraværshistorikkVirksomhet, summertSykefraværshistorikkBransjeEllerNæring);
     }
+
+    @GetMapping(value = "/{orgnr}/sykefravarshistorikk/legemeldtsykefravarsprosent")
+    public LegemeldtSykefraværsprosent hentLegemeldtSykefraværsprosent(
+            @PathVariable("orgnr") String orgnrStr,
+            HttpServletRequest request
+    ) {
+        InnloggetBruker bruker = tilgangskontrollService.hentInnloggetBruker();
+
+        tilgangskontrollService.sjekkTilgangTilOrgnrOgLoggSikkerhetshendelse(
+                new Orgnr(orgnrStr),
+                bruker,
+                request.getMethod(),
+                "" + request.getRequestURL()
+        );
+        Underenhet underenhet = enhetsregisteretClient.hentInformasjonOmUnderenhet(new Orgnr(orgnrStr));
+
+        LegemeldtSykefraværsprosent legemeldtSykefraværsprosent =
```

Comment:
burde vi ikke supplere tilbake til brukeren hvilke kvartaler vi summerer for, og hvilket orgnummer beregningen gjøres på vegne av?